OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverStatsLogic.cs

```diff
@@ -344,10 +344,11 @@ ScrollItemWidget EconomyStats(Player player)
 
 			var res = player.PlayerActor.Trait<PlayerResources>();
 			var stats = player.PlayerActor.TraitOrDefault<PlayerStatistics>();
-			if (stats == null) return template;
+			if (stats == null)
+				return template;
 
 			template.Get<LabelWidget>("CASH").GetText = () => "$" + (res.Cash + res.Resources);
-			template.Get<LabelWidget>("EARNED_MIN").GetText = () => AverageEarnedPerMinute(res.Earned);
+			template.Get<LabelWidget>("EARNED_MIN").GetText = () => "$" + stats.DisplayIncome;
```

Comment:
Can we please rename the widget ID to `INCOME`? The header label should change to Income too.